src/client/engine/index.ts

```diff
@@ -0,0 +1,20 @@
+import { AxiosInstance } from "axios";
+
+import { IEngineApi } from "./interfaces";
+import { TGetEngineDescription } from "./requestTypes/engineDescription";
+import { TGetEngines } from "./requestTypes/engines";
+
+export default class EngineApi implements IEngineApi {
+	private api: AxiosInstance;
+
+	constructor(api: AxiosInstance) {
+		this.api = api;
+	}
+
+	public getEngines: TGetEngines = async () => {
+		return await this.api.get("/engines");
+	};
+	public getEngineDescription: TGetEngineDescription = async (args) => {
```

Comment:
По возможности, сразу деструктуризируй
```suggestion
	public getEngineDescription: TGetEngineDescription = async ({ engine }) => {
```